src/components/Buy/InventoryTable.jsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect } from 'react';
+import { collection, getDocs, getDoc } from 'firebase/firestore/lite';
+import { MDBDataTable } from 'mdbreact';
+import { db } from '../../firebase/firebase_config';
+
+import { buyColumn, noBuyColumns } from './column';
+import './InventoryTable.css';
+
+function InventoryTable({ buyFunctionality, tableData, setTableData, handleAddToCart }) {
+  // This function builds the data we want to display in the inventory
+  // it builds this through book the current textbook information and getting
+  // information about the seller (for the payment methods accepted)
+
+  async function tableFormatBook(book) {
+    const bookData = book.data();
+    const { seller } = bookData;
+    let maybeSellerPaymentMethods = '';
+    // Get the payment methods info
+    try {
+      const maybeSeller = await getDoc(seller);
+      const maybeSellerData = maybeSeller.data();
+      if (maybeSellerData.paymentMethod.venmo) {
+        maybeSellerPaymentMethods += 'Venmo';
+      } else if (maybeSellerData.paymentMethod.cash) {
+        maybeSellerPaymentMethods += 'Cash';
+      }
```

Comment:
are we supposed to only show one of their methods? Is the product requirement that we just show venmo if the user has both options available?

Reply:
Is this specified in the figma? I think it just makes more sense to show both payment methods if they are there

Reply:
yea i agree. In that case, you want something like this

```suggestion
      if (maybeSellerData.paymentMethod.venmo) {
        maybeSellerPaymentMethods += 'Venmo ';
      } 
      if (maybeSellerData.paymentMethod.cash) {
        maybeSellerPaymentMethods += 'Cash';
      }
```

Reply:
added a space after venmo so they arent squished together